jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java

```diff
@@ -959,23 +1003,38 @@ public ContentChannel handleRequest(final Request request, final ResponseHandler
     }
 
     private static class ParameterPrinterRequestHandler extends AbstractRequestHandler {
+
         private static final CompletionHandler NULL_COMPLETION_HANDLER = null;
 
         @Override
-        public ContentChannel handleRequest(final Request request, final ResponseHandler handler) {
-            final Map<String, List<String>> parameters =
-                    new TreeMap<>(((HttpRequest)request).parameters());
-            final ContentChannel responseContentChannel
-                    = ResponseDispatch.newInstance(Response.Status.OK).connect(handler);
-            responseContentChannel.write(
-                    ByteBuffer.wrap(parameters.toString().getBytes(StandardCharsets.UTF_8)),
-                    NULL_COMPLETION_HANDLER);
+        public ContentChannel handleRequest(Request request, ResponseHandler handler) {
+            Map<String, List<String>> parameters = new TreeMap<>(((HttpRequest)request).parameters());
+            ContentChannel responseContentChannel = ResponseDispatch.newInstance(Response.Status.OK).connect(handler);
+            responseContentChannel.write(ByteBuffer.wrap(parameters.toString().getBytes(StandardCharsets.UTF_8)),
+                                         NULL_COMPLETION_HANDLER);
 
             // Have the request content written back to the response.
             return responseContentChannel;
         }
     }
 
+    private static class RequestTypeHandler extends AbstractRequestHandler {
```

Comment:
I cannot find use of this class outside unit tests

Reply:
It's in a test class so I hope not :-)

Reply:
Oops, my fault